### PR TITLE
Upgrade intracom link and background

### PR DESCRIPTION
**Pull request by OwOthmane**
Stan Cassie

### CODE REVIEW

**Comment by CL0Pinette** [comment]
CHAD !

File: _data/links.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@
     - 'background-size: 90%'
 
   - name: Intracom
-    href: http://intracom.epita.fr/
+    href: https://intracom-new.epita.fr/
```
Comment (matiboux) [inline _data/links.yml:20]:
```suggestion
    href: https://intracom.epita.fr/
```

**Review by matiboux — CHANGES_REQUESTED** [review]
Seems like the Platform was moved to https://intracom.epita.fr/.
`intracom-new.epita.fr` shows `404 page not found`.

So the pull request seems outdated to me.

However, the pull request would be still useful for upgrading the link to `https` (though not required). I'll propose changes for that.

Tell me if I'm wrong! Otherwise I'll merge changes in, let's say, 20 hours or so.

**Review by matiboux — APPROVED** [review]
There! Upgraded the link to https but also updated the background image.
Thanks for the PR.

**Pull request merged**